### PR TITLE
add extraVolumes and extraVolumeMounts

### PR DESCRIPTION
**Pull request by konvergence**

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-appVersion: "1.26.1"
+appVersion: "1.27.2"
 description: Helm chart for fusionauth
 type: application
 name: fusionauth
-version: 0.0.0+dev
+version: 0.9.3
```
Comment (drpebcak) [inline chart/Chart.yaml:6]: Can you change this back? The chart here is not supposed to have a specific version number in it.

File: chart/values.yaml [hunk 2]
```diff
@@ -224,4 +225,15 @@ startupProbe:
     port: http
   failureThreshold: 20
   periodSeconds: 10
-  timeoutSeconds: 5
\ No newline at end of file
+  timeoutSeconds: 5
+
+
+
```
Comment (drpebcak) [inline chart/values.yaml:231]: Can you remove these extra newlines? There should only be max 1 empty line

File: chart/values.yaml [hunk 3]
```diff
@@ -224,4 +225,15 @@ startupProbe:
...
+extraVolumes: []
+##  - name: custom-css-data
+##    persistentVolumeClaim:
+##      claimName: custom-css-data
+
+extraVolumeMounts: []
```
Comment (drpebcak) [inline chart/values.yaml:237]: For these two new values, can you add comments in the format used above the other values? These are used to generate the table in the readme.

File: chart/templates/deployment.yaml [hunk 4]
```diff
@@ -108,22 +108,43 @@ spec:
...
+      {{- end }}
+
+      {{- if .Values.kickstart.enabled }}
         - name: {{ template "fusionauth.fullname" . }}-config-volume
           configMap:
             name: {{ template "fusionauth.fullname" . }}-kickstart-config
       {{- end }}
+
+
+
```
Comment (drpebcak) [inline chart/templates/deployment.yaml:147]: Can you remove these blank newlines?

File: chart/templates/deployment.yaml [hunk 5]
```diff
@@ -108,22 +108,43 @@ spec:
...
+          {{- if .Values.kickstart.enabled }}
             - name: {{ template "fusionauth.fullname" . }}-config-volume
               mountPath: /kickstart
           {{- end }}
+
+          {{- if .Values.extraVolumeMounts }}
+          {{- toYaml .Values.extraVolumeMounts | nindent 12 }}
+          {{- end }}
+
+
```
Comment (drpebcak) [inline chart/templates/deployment.yaml:125]: Can you remove these blank newlines?

**Review by drpebcak — CHANGES_REQUESTED** [review]

File: chart/Chart.yaml (hunk 1 above)
Comment (konvergence) [reply chart/Chart.yaml]: done on 78bc8a5

File: chart/values.yaml (hunk 2 above)
Comment (konvergence) [reply chart/values.yaml]: done on 78bc8a5

File: chart/values.yaml (hunk 3 above)
Comment (konvergence) [reply chart/values.yaml:237]: I have added the comments. But I don't know how to add sample as comment for readme

File: chart/templates/deployment.yaml (hunk 4 above)
Comment (konvergence) [reply chart/templates/deployment.yaml]: done on 78bc8a5

File: chart/templates/deployment.yaml (hunk 5 above)
Comment (konvergence) [reply chart/templates/deployment.yaml]: done on 78bc8a5

File: chart/values.yaml (hunk 3 above)
Comment (drpebcak) [reply chart/values.yaml:237]: That's fine, I can generate it after this is merged. Can you fix the extra space between 'each' and 'extraVolumes'?

Comment (konvergence) [reply chart/values.yaml:237]: see 560654d

Hi, 
i thinks, we need to give samples in commnents for extraVolumes and extraVolumeMounts

**Review by drpebcak — APPROVED** [review]

**Pull request merged**